src/portal_visualization/builders/anndata_builders.py

```diff
@@ -239,3 +280,142 @@ def _add_spatial_view(self, dataset, vc):
         )
         spatial.use_coordination(cells_layer)
         return spatial
+
+
+class SpatialMultiomicAnnDataZarrViewConfBuilder(SpatialRNASeqAnnDataZarrViewConfBuilder):
+    """
+    Wrapper class for creating a AnnData-backed view configuration for multiomic spatial data
+    such as Visium.
+    """
+
+    def __init__(self, entity, groups_token, assets_endpoint, **kwargs):
+        super().__init__(entity, groups_token, assets_endpoint, **kwargs)
+        self._scatterplot_w = 3
+        self._spatial_w = 3
+
+    def _get_scale_factor(self):
+        z = self.zarr_store
+        visium_scalefactor_path = 'spatial/visium/scalefactors/spot_diameter_fullres'
+        if visium_scalefactor_path in z['uns']:
+            return z['uns'][visium_scalefactor_path][()].tolist()
+
+    def _set_up_dataset(self, vc):
+        adata_url = self._build_assets_url(
+            'hubmap_ui/anndata-zarr/secondary_analysis.zarr', use_token=False)
+        image_url = self._build_assets_url(
+            'ometiff-pyramids/visium_histology_hires_pyramid.ome.tif', use_token=True)
+        # Add dataset with Visium image and secondary analysis anndata
+        visium_image = ImageOmeTiffWrapper(
+            img_url=image_url,
+            uid="visium",
+            request_init=self._get_request_init(),
+            coordination_values={
+                "fileUid": "visium"
+            }
+        )
+        visium_spots = AnnDataWrapper(
+            adata_url=adata_url,
+            obs_feature_matrix_path="X",
+            obs_spots_path="obsm/X_spatial",
+            obs_labels_names=self._obs_labels_names,
+            obs_labels_paths=self._obs_labels_paths,
+            feature_labels_path="var/hugo_symbol",
+            request_init=self._get_request_init(),
+            coordination_values={
+                "obsType": "spot",
+                "featureType": "gene",
+                "featureLabelsType": "gene",
+            }
+        )
+        obs_sets = AnnDataWrapper(
```

Comment:
```suggestion
        visium_spots = AnnDataWrapper(
```

I think we can combine these into a single AnnDataWrapper instance